canonical-websites/www.ubuntu.com
# Remove /third-parties page

## Done

- Removed 3rd parties page
- Updated link in /legal/developer-terms-and-conditions/2016-04-06

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that http://0.0.0.0:8001/legal/data-privacy/third-parties has been removed and redirected
- See that http://0.0.0.0:8001/legal/developer-terms-and-conditions/2016-04-06 has a new dev link

## Issue / Card

Fixes #4657
